### ISSUE TITLE
Usage with vim-sneak

### ISSUE THREAD

**Issue by simonweil** [svermeulen/vim-easyclip#60]
Hi,
Is use justinmk/vim-sneak for quite a while and I'm used to using `s` for the sneak motions.
How would I map the `s` in `easyclip` to some other key?
Thanks,
Simon

**Comment by svermeulen** [comment]
This should work fine, since the mapping for `s` is disabled by default with easy clip.  The options section of the ReadMe covers how to map different keys to different operations.  For example, if you want to use `gs` for substitute, include this in your vimrc:

    nmap <silent> gs <plug>SubstituteOverMotionMap
    nmap gss <plug>SubstituteLine
    xmap gs <plug>XEasyClipPaste

**Comment by simonweil** [comment]
Oh, OK thanks, I use (neo)vim, but I'm not really a pro in VimL and the configuration.
I'll reopen if I have any issues.
Thank again :)

**Comment by svermeulen** [comment]
There's some outstanding neovim specific issues (eg: #55) so feel free to log any other ones you see.  I need to upgrade to neovim and take a look at these soon.

**Comment by simonweil** [comment]
I installed the plugin and the `s` mapping is overridden by `easyclip` although I didn't enable it.

Typing `:map s` gives me:
```viml
s  s           * <C-O>"_cs
x  s           * "_s
n  s           * "_s
```
So it isn't setting the substitute commands but still remapping to vim's `s` with the blackhole which I really want remapped to `sneak`'s mappings. This is what I expect to see:
```viml
x  s             <Plug>Sneak_s
n  s             <Plug>Sneak_s
```

Please help me get my `sneak` mapping back with `easyclip` installed. :)

**Comment by svermeulen** [comment]
Yes I see what you mean.  Easyclip will only override the `s` mappings if it isn't already mapped to something.  I've noticed that vim-sneak does the same thing, so I think the easiest way to fix this would be to initialize vim-sneak before vim-easyclip, so that it's mappings take precedence.

If you're using NeoBundle, this is just a matter of placing these lines in this order:

    NeoBundle 'justinmk/vim-sneak'
    NeoBundle 'svermeulen/vim-easyclip.git'

**Comment by simonweil** [comment]
Well this solved it for me but I would rather have a more elegant solution such as an option to disable the `s` mapping.
Would you accept such a pull request?

**Comment by svermeulen** [comment]
Sure.

If you look at `BlackHole.vim` you can see what it's doing there.  You could set `g:EasyClipEnableBlackHoleRedirectForChangeOperator` to `0` but then you'd still have to remap all the other change keys so that's not great.  I don't think having a boolean flag for each specific key is very good either, so maybe instead we could provide an easy clip setting for a list of bindings to leave alone.

This is really only a problem when two plugins disagree on a mapping, since if there's a vimrc mapping then easy clip will always respect that.  But still could be useful.
